css/properties/word-break.json

```diff
@@ -115,6 +115,30 @@
               "deprecated": false
             }
           }
+        },
+        "auto-phrase": {
+          "__compat": {
+            "description": "<code>auto-phrase</code>",
+            "support": {
+              "chrome": {
+                "version_added": "119",
+                "notes": "Only works if Japanese is declared with lang=\"ja\" and does not affect other languages."
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror"
```

Comment:
All other browsers must also be added, including Safari, Samsung Internet, etc.

Reply:
I've added them now.